src/OpenTelemetry.AutoInstrumentation.StartupHook/Logger.cs

```diff
@@ -20,7 +20,7 @@ namespace OpenTelemetry.AutoInstrumentation.StartupHook;
 
 internal static class Logger
 {
-    private static readonly IOtelLogger Log = OtelLogging.GetLogger();
+    private static readonly IOtelLogger Log = OtelLogging.GetLogger("StartupHook");
```

Comment:
`nameof(StartupHook) `?

Reply:
That actually is more of a mock name for now, I would like to hear from others what the fitting suffix is in their opinion.